### DISCUSSION TITLE
Telegram Icon

### DISCUSSION THREAD

**Discussion by abdullah-cse** [lucide-icons/lucide#1466] [Ideas]
I ❤ and heavily use [Lucide-Icon](https://pub.dev/packages/lucide_icons).
Just for a single [Telegram Icon](https://telegram.org/), i don't like to switch [FontAwesome Icon](https://fontawesome.com/icons).

So, it would be nice if there were a [Telegram Icon](https://telegram.org/)

**Comment by karsa-mistmere** [comment]
We will not be adding any new brand icons for various reasons, see discussions under #670 #94 #552 
We recommend that people use simpleicons.org in addition to Lucide, as both sets use a 24 by 24 px grid and simpleicons is dedicated solely to brand icons.
